smoke/features/data/sharesecret.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
$ shellcheck sharesecret.sh 

In sharesecret.sh line 17:
create_shared_secret ${1}
                     ^--^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
create_shared_secret "${1}"

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...